opencti-platform/opencti-graphql/src/utils/syntax.js

```diff
@@ -135,7 +135,7 @@ export const cleanupIndicatorPattern = (patternType, pattern) => {
 };
 
 export const systemChecker = /^\d{0,10}$/;
-export const domainChecker = /^(|(([\p{L}\d]|[\p{L}\d][\p{L}\d\p{Pd}_]*[\p{L}\d])\.)*([\p{L}\d]|[\p{L}\d][\p{L}\d\p{Pd}]*[\p{L}\d]))$/u;
+export const domainChecker = /^(?=.{1,253}$)(?!-)(?!_)([a-zA-Z0-9-_À-ÿ]{1,63}(\.[a-zA-Z0-9-_À-ÿ]{2,63})*)$/;
```

Comment:
This regex maybe fix failing test and fix perf issues, let me know if it works better for you 😃 
```suggestion
export const domainChecker = /^(?=.{1,253}$)(?!-)(?!.*--)([a-zA-Z0-9-_À-ÿ]{1,63}\.)+([a-zA-Z]{2,63})$/;
```
Should this regex match this domain : `invalid_domain.com` ? If yes maybe we could add a test to cover this case